### PR TITLE
feat: Add horitzontal align prop to ma text

### PR DESCRIPTION
**Pull request by carlotamolintr**

### CODE REVIEW

File: vetur/attributes.json [hunk 1]
```diff
@@ -413,6 +413,11 @@
       "gray"
     ]
   },
+  "ma-text/align": {
+    "type": "string",
+    "description": "Set an horitzontal alignment to the text",
```
Comment (afontcu) [inline vetur/attributes.json:418]:
```suggestion
    "description": "Set an horitzontal alignment to the text. Only applies to block-level elements",
```

File: src/components/MaText/MaText.stories.js [hunk 2]
```diff
@@ -32,9 +38,10 @@ export default {
 
 const Template = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
-  template: `<div style="display: flex;">
+  template: `
+  <div style="width: 20rem">
```
Comment (afontcu) [inline src/components/MaText/MaText.stories.js:42]: I'd keep the flex layout for now, so we can easily see the text without the blueish background

```suggestion
  <div style="display: flex;width: 20rem">
```

**Review by afontcu — APPROVED** [review]
LGTM! 🚀  I've added a couple of small suggestions

**Comment by afontcu** [comment]
Remember that this PR is an addition, so its title (and thus the merge commit) should start with `feat: ` to create a new release

**Review by edufarre — COMMENTED** [review]
Shall we add a test that ensures that given an `align` prop value, the expected alignment style is set?

**Comment by carlotamolintr** [comment]
> Shall we add a test that ensures that given an `align` prop value, the expected alignment style is set?

Sure! I just did it. 🙂

**Review by edufarre — APPROVED** [review]

**Pull request merged**